sale_update_pricelist/README.rst

```diff
@@ -0,0 +1,30 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Sale: Update pricelist
+======================
+
+With this module, when a pricelist is changed in the sales order, all the prices in the lines are updated with the ones defined in the pricelist.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Hugo Santos <hugo.santos@factorlibre.com>
+
+Maintainer
```

Comment:
2015-08-21 6:21 GMT-05:00 Hugo Santos <notifications@github.com>:

> @nhomar <https://github.com/nhomar> How can I add the runbot link?
>
> Is this flag? [image: Runbot Status]
> <https://runbot.odoo-community.org/runbot/repo/github-com-oca-sale-workflow-167>
>

here the new template:

https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

The variables can be taken from the runbot url itself (sorry I don't
remember where is exactly the table with all the mapping of IDs.






-- 
--------------------
Saludos Cordiales

CEO at Vauxoo <https://www.vauxoo.com> Odoo's Gold Partner.

[image: --]
Nhomar Hernandez
[image: http://]about.me/nhomar
<http://about.me/nhomar?promo=email_sig>